source/install-by-copying-files/index.html.md.erb

```diff
@@ -0,0 +1,73 @@
+---
+title: Install by copying files
+weight: 10
+---
+
+# Install by copying files
+
+You can install GOV.UK Frontend by copying our CSS and JavaScript files into your project.
+
+<%= warning_text('We recommend <a href="/installing-with-npm/">installing with Node.js package manager (npm)</a> instead.
+') %>
+
+## What you cannot do
+
+If you install this way, you will not be able to:
+
+- change settings, for example you will not be able to override colours or set your own font
+- use the Nunjucks macros from the [Design System website](https://design-system.service.gov.uk/)
+- import an individual component’s CSS or JavaScript
+- use GOV.UK Frontend's colours or mixins in your custom code
+
+## Copy the files
+
+1. Download the [`dist` folder and its subfolders](https://github.com/alphagov/govuk-frontend/tree/master/dist) from the GOV.UK Frontend GitHub repo.
+2. Copy the 2 `.css` files to `<YOUR-APP>/stylesheets`.
+3. Copy the `.js` file to `<YOUR-APP>/javascript`.
+4. Copy the `assets` folder to the root of your project’s public folder, so that for example `<YOUR-SITE-URL>/assets/govuk-logotype-crown.png` will work in your users’ browsers.
+
+## Check it works
+
+1. Create a page in your project using the following HTML:
+
+    ```html
+    <!DOCTYPE html>
+    <html lang="en" class="govuk-template ">
+      <head>
+        <title>Example if you’ve installed GOV.UK Frontend by copying files</title>
```

Comment:
Should this use something that fits the Page Title we want people to use?

```suggestion
        <title>Example - GOV.UK</title>
```

Reply:
👍

Reply:
Still pending, I think.

Reply:
Oh sorry I misunderstood your comment - will fix.